include/psa/crypto_values.h

```diff
@@ -1187,7 +1134,7 @@
  */
 #define PSA_ALG_IS_HASH_AND_SIGN(alg)                                   \
     (PSA_ALG_IS_RSA_PSS(alg) || PSA_ALG_IS_RSA_PKCS1V15_SIGN(alg) ||    \
-     PSA_ALG_IS_DSA(alg) || PSA_ALG_IS_ECDSA(alg))
+     PSA_ALG_IS_ECDSA(alg))
```

Comment:
Do we need to provide an implementation specific `PSA_ALG_IS_HASH_AND_SIGN()` that understands `PSA_ALG_IS_DSA()`?

Reply:
Yes, if we implement DSA, which we don't.

The current tooling doesn't allow us to document a macro without giving a definition for it. When we move away from Doxygen we'll be able to do it.

Reply:
But I actually did preserve the definition with DSA support in `crypto_extra.h`, anyway.